File: app/src/main/java/com/example/testeandroidv2/Service/LoginService.java
package com.example.testeandroidv2.Service;

import com.google.gson.JsonObject;

import retrofit.Call;
import retrofit.http.Field;
import retrofit.http.FormUrlEncoded;
import retrofit.http.POST;

public interface LoginService {

    @FormUrlEncoded
    @POST("login")
    Call<JsonObject> authentication(@Field("user") String user, @Field("password") String password);
}

File: app/src/main/java/com/example/testeandroidv2/service/LoginService.kt
package com.example.testeandroidv2.service

import com.example.testeandroidv2.model.login.LoginBody
import com.example.testeandroidv2.model.login.LoginResponse
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface LoginService {

    @POST("requestLogin")
    fun login(@Body loginBody: LoginBody): Call<LoginResponse>?
}